glide.yaml

```diff
@@ -13,9 +13,10 @@ import:
 - package: github.com/johanneswuerbach/nfsexports
 - package: github.com/mitchellh/go-ps
 - package: github.com/zchee/go-qcow2
-- package: github.com/zchee/libhyperkit
 - package: github.com/jceel/lib9p
   version: 7be3e0b914fbb2d2ecfe2984d02bacfd8e762eae
+- package: github.com/zchee/go-vmnet
+- package: github.com/moby/hyperkit
```

Comment:
I actually need 62fb993cc65090d3efe29e9b1ed871b017eeaa4d, it contains a PR I submitted yesterday to add support for qcow and sparsebundle disks. What's the best way to pin to that in glide?